tests/Aquifer.Common.IntegrationTests/Utilities/VersificationUtilitiesTests.cs

```diff
@@ -0,0 +1,133 @@
+﻿using Aquifer.Common.Services.Caching;
+using Aquifer.Common.Utilities;
+
+namespace Aquifer.Common.IntegrationTests.Utilities;
+
+public sealed class VersificationUtilitiesTests(App _app) : TestBase<App>
+{
+    private readonly ICachingVersificationService _versificationService = _app.GetRequiredService<ICachingVersificationService>();
```

Comment:
Per the PR description, this amount of manual DI is acceptable to me because getting class constructor level DI injection with third-party packages isn't worth it.